### ISSUE TITLE
downloading speed is zero

### ISSUE THREAD

**Issue by stingwan** [linuxserver/docker-qbittorrent#69]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

If you are new to Docker or this application our issue tracker is **ONLY** used for reporting bugs or requesting features. Please use [our discord server](https://discord.gg/YWrKVTn) for general support.

------------------------------

## Current Behavior

after add magnet link， the speed is always 0. qbittorent can find peers ,and the "Tracker status" is working ,But the download speed is always 0.       )..(
## Steps to Reproduce

1.
2.
3.
4.

## Environment
**OS:**      
**CPU architecture:** x86_64/arm32/arm64      
**How docker service was installed:**

**Comment by j0nnymoe** [comment]
Please could you provide the information we've requested in the template.
Also please test using this link: magnet:?xt=urn:btih:54da0b79719064aa10fe2cc4e13630a1222d1939&dn=archlinux-2020.03.01-x86_64.iso&tr=udp://tracker.archlinux.org:6969&tr=http://tracker.archlinux.org:6969/announce

**Comment by richardrey** [comment]
I have the same problem, but from what I have been able to test it is a problem in how it saves the new config, since I have made many deployments, in all it works perfectly when everything is by default, without changing a single parameter.

The problems start when any section of the configuration is modified, I have activated the directory of incomplete downloads and the downloads are set to 0, both new and those that were before the configuration change.

I have checked and it is not only the change of directory but almost any section, it happens to me when I put the list of trackers.

This is the simplest deployment mode I use:

`docker run -dit --name QBittorrent -e PUID=1000 -e PGID=1000 -e TZ=Europe/Madrid -e WEBUI_PORT=9091 -p 6881:6881 -p 6881:6881/udp -p 9091:9091 -v config:/config -v $PWD/descargas:/downloads -v descargas_incompletas:/downloads/incomplete --restart always linuxserver/qbittorrent`

Greetings
